static/app/views/organizationCreate/index.tsx

```diff
@@ -9,15 +12,62 @@ import ConfigStore from 'sentry/stores/configStore';
 import {OrganizationSummary} from 'sentry/types';
 import {normalizeUrl} from 'sentry/utils/withDomainRequired';
 
+enum RegionDisplayName {
+  US = '🇺🇸 United States of America (US)',
+  DE = '🇪🇺 European Union (EU)',
+}
+
+function getRegionChoices(): [string, string][] {
+  const regions = ConfigStore.get('regions') ?? [];
+
+  return regions.map(({name, url}) => {
+    const regionName = name.toUpperCase();
+    if (RegionDisplayName[regionName]) {
+      return [url, RegionDisplayName[regionName]];
+    }
+
+    return [url, name];
```

Comment:
If we're sending the host name for the region in the request, do we parse the URL in the server side? Should we be sending the region name instead so that we don't have to parse URLs from user data?

Reply:
As far as I know, the region is unused/extraneous in the request body, but I couldn't figure out how to omit it from the form data 😅 Ideally we wouldn't send this at all.

Reply:
We could supply a `model` to `ApiForm` that omits the `region` field from the serialized request data via the `transformData` method.

https://github.com/getsentry/sentry/blob/563e619b1f96ba342487ad8e3332d9edfe03c4e3/static/app/components/forms/model.tsx#L324-L334

is the signature for `FormModel.transformData` and an existing usage can be found here 

https://github.com/getsentry/sentry/blob/563e619b1f96ba342487ad8e3332d9edfe03c4e3/static/app/components/modals/sentryAppPublishRequestModal.tsx#L44-L61